src/translations/en.json

```diff
@@ -1375,17 +1375,31 @@
             "migrate": "Migrate config"
           },
           "card": {
+            "alarm-panel": {
+              "name": "Alarm panel"
+            },
             "alarm_panel": {
```

Comment:
Duplicate `alarm-panel` and `alarm_panel`?

Reply:
There's an existing entry for `alarm_panel` which doesn't conform to the rest of the 'standard' in that json obj.

I could move the items and remove that one but would have to do a find and replace in the repo which will increase the risk somewhat.

Up to you guys what we should do 👍

Reply:
Let's rename it

Reply:
Done